syntax = "proto3";
option csharp_namespace = "HSR.MotionCapture.Net.Protos";

import "unityQuaternion.proto";

message FaceData {
    message BlendShapeData {
        string name = 1;
        float value = 2;
    }

    UnityQuaternion headRotation = 1;
    repeated BlendShapeData blendShapes = 2;
}